plugins/aws/resoto_plugin_aws/resource/sqs.py

```diff
@@ -20,6 +21,8 @@ class AwsSqsQueue(AwsResource):
     mapping: ClassVar[Dict[str, Bender]] = {
         "id": S("QueueName"),
         "name": S("QueueName"),
+        "ctime": S("CreatedTimestamp") >> F(lambda x: utc_str(datetime.fromtimestamp(x))),
```

Comment:
mmhh I would assume that the timestamp is based on UTC (guessing) - but it is interpreted with local time (fromtimestamp). maybe we should interpret this timestamp in UTC as well?